src/Common/src/System/Drawing/KnownColor.cs

```diff
@@ -7,6 +7,7 @@
 namespace System.Drawing
 {
     [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
```

Comment:
Are we continuing to build for netcoreapp20 here?

Reply:
Yes we are, https://github.com/dotnet/corefx/pull/31142#issuecomment-405980805